sdk/storage/storage-file/test/aborter.spec.ts

```diff
@@ -25,19 +25,14 @@ describe("Aborter", () => {
     recorder.stop();
   });
 
-  it("should set value and get value successfully", async () => {
-    const aborter = Aborter.none.withValue("mykey", "myvalue");
-    assert.deepStrictEqual(aborter.getValue("mykey"), "myvalue");
```

Comment:
Removed this test since withValue is removed from the new abort-controller package.